Formula/parmetis.rb

```diff
@@ -0,0 +1,72 @@
+class Parmetis < Formula
+  desc "MPI library for graph/mesh partitioning and fill-reducing orderings"
+  homepage "http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview"
+  url "http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/parmetis-4.0.3.tar.gz"
+  sha256 "f2d9a231b7cf97f1fee6e8c9663113ebf6c240d407d3c118c55b3633d6be6e5f"
+
+  # METIS 5.* is required. It comes bundled with ParMETIS.
+  # We prefer to brew it ourselves.
+  depends_on "metis"
+
+  depends_on "cmake" => :build
+  depends_on "open-mpi"
+
+  # Do not build the METIS 5.* that ships with ParMETIS.
+  patch :DATA
+
+  # Bug fixes from PETSc developers. Mirrored because the SHA-256s get
+  # invalidated every time Bitbucket updates the Git version they use.
+  patch do
+    # From: https://bitbucket.org/petsc/pkg-parmetis/commits/82409d68aa1d6cbc70740d0f35024aae17f7d5cb/raw/
```

Comment:
This was committed in 2012, how hasn't it made it into a stable release yet? 😅.

Reply:
My best guess is the student graduated, and the project didn't progress much after that.